doc/guide/Guide_Introduction.ipynb

```diff
@@ -36,13 +40,14 @@
     "$$\\text{severity} = F(\\text{hazard intensity}, \\text{exposure}, \\text{vulnerability}) = \\text{exposure} * f_{\\text{imp}}(\\text{hazard intensity})$$\n",
     "\n",
     "where $f_{\\text{imp}}$ is the impact function which parametrizes to what extent\n",
-    "an exposure will be affected by a specific hazard. While 'vulnerability\n",
-    "function' is broadly used in the modelers community, we refer to it as\n",
+    "an exposure will be affected by a specific hazard. **While the term 'vulnerability\n",
+    "function' is broadly used in the modelers community, we adopt the term\n",
```

Comment:
based on @chahank comment, I would modify the sentence "While the term 'vulnerability function' is broadly used in the modelers community, we adopt the term 'impact function' to explicitly include the option of opportunities (i.e. negative damages)" to something like "While the term 'vulnerability function' is broadly used in the modelers community, we adopt the broader term 'impact function'. Impact functions can be vulnerability functions or structural damage functions, but could also be productivity functions or warning levels. This definition also explicitly includes the option of opportunities (i.e. negative damages). "